src/leap/mail/smtp/tests/test_gateway.py

```diff
@@ -23,6 +23,7 @@
 import re
```

Comment:
Cool, let's merge and attack the tests. I haven't tested smtp beyond fixing its initialization yet.